src/main/webapp/WEB-INF/glassfish-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please remove this glassfish settings file since our Servelet Container for this project is Tomcat